woocommerce/wc-admin
# Add Revenue Report API/data-layer

Closes #224.

This PR adds the ability to fetch revenue reports from the new v3 endpoint. It stores results under a keyed 'queries' object. Based on the designs, it seems like we might have different queries for the chart, vs the current page of the table for example, so it will be important to store multiple queries.

To Test:
* Run `npm install` followed by `npm test` and make sure all tests pass.
* Pull the latest WooCommerce v3 API from `feature/rest-api-v3-reports` (you might need to deactivate `wc-api-dev`).
* You need to deactivate and reactivate WooCommerce to get the correct summary MySQL tables.
* Uncomment the two `withSelect` lines in `client/analytics/report/revenue.js`, and move the call into the compose.
* Open the revenue report and confirm a request is made and some data is returned. You can also use the Redux console, and select 'wc-admin' from the context dropdown there. You should see actions fire.
